main.c

```diff
@@ -72,6 +72,12 @@ int CalcInit(void)
     return 0;
 }
 
+/*A void function that wil print errors for the user*/
+void printError(char stringError[])
+{
+	PrintString("Error! : %s! The application will now exit...\n", stringError);
```

Comment:
I'd rather not have the exclamation point hard-coded in the string format. It's really not intuitive into doing so. Furthermore, could you please capitalize the first letter of the routine's name (just like every function here in the code)?